Enhanced Barter.user.js

```diff
@@ -146,6 +146,44 @@ function barterReady() {
                 },
             }));
         });
+    
+    // Add GGdeals for match screen
+    $(".matchcol li a").get()
+        .forEach((elem) => {
+
+        if (elem.href.includes("barter.vg/bundle/") || elem.href.includes("barter.vg/u/")) {
+            return;
+        }
+        let barterId = elem.href.match(/vg\/i\/\d+/g)[0].split("/")[2];
+
+        $(elem).parent()
+            .prepend(`
+                <span class="tag">
+                    <a style="cursor: pointer;" id="ggdeals_${barterId}">
+                        <img src="https://bartervg.com/imgs/ico/gg.png" width="18" height="18" title="GG.Deals: Click to load price info!">
+                    </a>
+                    <small id="ggdeals_${barterId}_after"></small>
+                </span>`);
+        $(`[id="ggdeals_${barterId}"]`).click(() => {
+            (async() => {
+                let barterItem = await getBarterItemInfo(barterId);
+                let type = barterItem.source_profile.includes("app") ? "app" : "sub";
+                let id = barterItem.sku;
+                return request({
+                    "method": "GET",
+                    "url":    `https://gg.deals/steam/${type}/${id}`,
+                    "onload": (response) => {
+                        let parser = new DOMParser();
+                        let body = parser.parseFromString(response.responseText, "text/html");
+                        let price = $("#game-header-current-prices .price", body).get()
```

Comment:
jquery doesn't need DOM, u can simply use a string
```suggestion
let price = $("#game-header-current-prices .price", response.responseText).get()
```

Reply:
Changing this creates a lot of errors in the console.
"Refused to load the image 'https://barter.vg/images/svgs/icons.svg?v=53e6e40d' because it violates the following Content Security Policy directive: "img-src data: https://bartervg.com https://steamcdn-a.akamaihd.net https://*.steamstatic.com".

I left it as it was.

Reply:
you can ignore that error

Reply:
Are you sure? It creates literally hundreds of it. I would rather keep the long form, but you own it, so if you prefer the shorter (and easier to read) but hundreds of warning creating version I will go with that of course.

Reply:
I know why it happens. It's because when jquery parses html string it tends to load all images, which is dumb. Anyway, leave it as it is. It's fine.